locator_node:
  ros__parameters:
    use_sim_time: False
    base_frame_id: "base_footprint"
    global_frame_id: ""  # If empty, frame_id of poses is used
    odom_frame_id: "odom"
    pose_topic: "/bridge_node/client_localization_pose/pose"
    transform_tolerance: 0.1

locator_node_rclcpp_node:
  ros__parameters:
    use_sim_time: False

map_server:
  ros__parameters:
    use_sim_time: False
    frame_id: "map"
    grid_resolution: 0.02
    measurement_contribution: 100
    topic_name_cloud: "/bridge_node/client_localization_map"
    topic_name_grid: "map"

map_server_rclcpp_node:
  ros__parameters:
    use_sim_time: False
